src/preset.ts

```diff
@@ -619,42 +619,48 @@ const referencedConfigurePresets: Map<string, Set<string>> = new Map();
  * They should NOT be used together.
  * They should Not call each other.
  */
-export function expandVendorForConfigurePresets(folder: string): void {
+export async function expandVendorForConfigurePresets(folder: string, sourceDir: string, workspaceFolder: string, errorHandler?: ExpansionErrorHandler): Promise<void> {
```

Comment:
Yeah I agree. Also, is there a reason you added `expandConfigurePreset` in there? Can I cut it back to where it was before without `expandConfigurePreset`

Reply:
The reason `expandConfigurePreset` was in there was because it needs to expand things like the generator and other things to ensure that the vendor can expand it's macros correctly, also accounting for inheritance. However, if we handle this in the overall method, I don't think we'll have to worry about it.